core/core.go

```diff
@@ -210,7 +206,7 @@ func NewTextile(ctx context.Context, conf Config) (*Textile, error) {
 	if err != nil {
 		return nil, err
 	}
-	t.bucks, err = tdb.NewBuckets(t.th, t.powc, t.collections.FFSInstances, conf.FFSDefaultConfig)
```

Comment:
Bucket archives just use the FFS instance default `StorageConfig`. We may want to provide an api to control that later

Reply:
Uhm, OK! I think the default is `repFactor: 1` which is just what we need for new nerpa since there aren't many miners. Before was 10 because `testnet` had a bunch.

A sensible default is a pain... if too high will fail to find enough miners. If too low, might choose "the highest reputation miner" but maybe isn't the best, so would fail.

Definitely allowing an API to customize which config sounds good. At least to say: "hey, the default isn't really helping now. Set a storageconfig that forces miner X".

Sharing thoughts.

Reply:
Agreed. I see plenty of FFS-related api enhancements to the Hub/Buckets API in our future, so we'll definitely have time to address this.